src/components/CommentsList.jsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react"
+import { useParams } from "react-router-dom"
+import * as api from "../api"
+import CommentsCard from "./CommentsCard"
+
+const CommentsList = () => {
+    const { article_id } = useParams();
+    const [comments, setComments] = useState([]);
```

Comment:
This looks like a great component to handle the comments list, it's a shame it's not being used inside _SingleArticle_ as far as I can tell